src/investments/registered-retirement-income-fund.ts

```diff
@@ -12,6 +12,14 @@ export interface RegisteredRetirementIncomeFund {
     MIN_WITHDRAWAL_PCT: { [K: number]: number };
 }
 
+export function getMinimumWithdrawalPercentage(age: number): number {
+    if (age >= 71) {
+        return RRIF.MIN_WITHDRAWAL_PCT[age];
+    }
+
+    return 1 / (90 - age);
```

Comment:
Tu peux utiliser une opération ternaire si tu veux raccourcir ça
```suggestion
    return age >= 71 ? RRIF.MIN_WITHDRAWAL_PCT[age] : 1 / (90 - age);
```

Reply:
Yes bonne idée, thanks! 👍